src/components/definition-list/dl.d.ts

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import { Dl } from '../definition-list';
+
+export interface DlProps {
+    /** This string will sepcify the text align styling of the `<dt></dt>`. */
+    dtTextAlign?: 'left' | 'center' | 'right';
+
+    /** This string will sepcify the text align styling of the `<dd></dd>`. */
+    ddTextAlign?: 'left' | 'center' | 'right';
+
+    /** This value will sepcify the width of the `StyledDtDiv` as a percentage. */
+    w?: number;
+
+    /** prop to render children. */
+    children: React.ReactNode;
```

Comment:
This isn't required in `dl.component`

Reply:
Yeah. It should be a required prop. It needs `.isRequired` adding in the dl.component file to match what is declared in the this file.

Reply:
✅